discord/cog.py

```diff
@@ -213,6 +220,8 @@ def __new__(cls: Type[CogMeta], *args: Any, **kwargs: Any) -> CogMeta:
 
         # Update the Command instances dynamically as well
         for command in new_cls.__cog_commands__:
+            if isinstance(command, ApplicationCommand) and command.guild_ids is None and len(new_cls.guild_ids) != 0:
+                command.guild_ids = new_cls.guild_ids
```

Comment:
shouldnt this be `__cog_guild_ids__`

Reply:
Have you checked the definition of `new_cls`

Reply:
`new_cls = super().__new__(cls, name, bases, attrs, **kwargs)` - everywhere else uses the dunder attributes

Reply:
![image of py-cord file](https://cdn.tooty.xyz/ssZO) - ignore all the other errors as i just copied raw text but heres a screenshot

Reply:
I tested before changing and missed that, thanks